src/meta/MetaServiceUtils.h

```diff
@@ -20,29 +20,73 @@ class MetaServiceUtils final {
  public:
   MetaServiceUtils() = delete;
 
+  /**
+   * @brief change column definition in memory
+   *
+   * @param cols existed column in schema
+   * @param prop schema property, mainly used to check if the col is ttl_col
+   * @param col
```

Comment:
This method is really hard to understand, I have to read the code every time 🤣 
Maybe add some description, about this function, e.g. `col` is the new column definition, `op` is the ADD/DROP/ALTER.

Reply:
done